src/test/kotlin/arrow/proofs/typeclasses/GivenTests.kt

```diff
@@ -1,17 +1,15 @@
-package arrow.proofs.typeclasses
+// TODO: It doesn't support a composite package
+package singlepackage
 
-import arrow.Given
 import org.junit.Test
 
 class GivenTest {
 
-    fun <A> given(evidence: @Given A = arrow.given): A =
-        evidence
-    @Given val x = "yes!"
+    // Arrow Meta Compiler Plugin is just applied during production code compilation (src/main/kotlin).
```

Comment:
I don't understand this, can you please expand as to what should change in other tests with an example?